lib/rules/_base.js

```diff
@@ -50,6 +50,9 @@ export default class {
     this.results = [];
     this.mode = options.shouldFix ? 'fix' : 'report';
 
+    this.workingDir = options.workingDir;
+    // TODO: add a deprecation for accessing _filePath
+    this.filePath = this._filePath = options.filePath;
```

Comment:
why move these?

Reply:
I see, sorry missed the other comment https://github.com/ember-template-lint/ember-template-lint/pull/2483#discussion_r859214553